### ISSUE TITLE
[$1000] [HOLD for payment 2023-01-25] `You closed this report` message in the archived room shows up in two lines

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#13839]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Actions Performed:

1. Go to Settings > Workspaces > New workspace
2. Click on the three vertical dots icon
3. Click Delete workspace
4. Click Delete
5. Go to that workspace's archived room

## Expected Result:
The message shouldn't be visible.

## Actual Result:
`You closed this report` message is displayed in two lines

## Workaround:
unknown
## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.2.43-1
**Reproducible in staging?:** y 
**Reproducible in production?:** y
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 
<img width="1050" alt="Screenshot 2022-12-27 at 12 27 00 PM" src="https://user-images.githubusercontent.com/43996225/209700259-3a603a3a-e824-4290-9b89-cad126eb1f5d.png">

<img width="960" alt="Untitled" src="https://user-images.githubusercontent.com/43996225/209700271-ad6dbded-6481-4495-bb51-94539521b2a9.png">

**Expensify/Expensify Issue URL:**
**Issue reported by:** @adeel0202 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1672126492151949

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01f2c041cf3af5ec83</li>
        <li>Upwork Job ID: 1607907836692803584</li>
        <li>Last Price Increase: 2023-02-14</li>
    </ul>
</details>

**Comment by marcochavezf** [comment]
Did a quick search and text comes from the server, grabbing this one.

**Comment by marcochavezf** [comment]
Posted [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1672259353511219) what could be the potential solution, and the best thing to do here is to ignore the closed action since the footer already explains why the workspace chat was closed. I'm going to create a PR shortly.

**Comment by marcochavezf** [comment]
PR to ignore the closed action [here](https://github.com/Expensify/App/pull/13887).

**Comment by marcochavezf** [comment]
Updating the expected result to be in-line with the PR ([slack convo](https://expensify.slack.com/archives/C02HWMSMZEC/p1673629956230669?thread_ts=1673613698.950449&cid=C02HWMSMZEC)).

**Comment by marcochavezf** [comment]
Not overdue, on hold for payment

**Comment by davidcardoza** [comment]
@marcochavezf Can you point me to who payment is due for?

**Comment by MelvinBot** [comment]
@davidcardoza, @marcochavezf 6 days overdue. This is scarier than being forced to listen to Vogon poetry!

**Comment by davidcardoza** [comment]
@parasharrajat offer sent.

**Comment by davidcardoza** [comment]
@parasharrajat Updated payment. 
@adeel0202 Contract sent

**Comment by MelvinBot** [comment]
Current assignee @davidcardoza is eligible for the External assigner, not assigning anyone new.

**Comment by MelvinBot** [comment]
Current assignee @parasharrajat is eligible for the External assigner, not assigning anyone new.

**Comment by MelvinBot** [comment]
Current assignee @marcochavezf is eligible for the External assigner, not assigning anyone new.

**Comment by davidcardoza** [comment]
### Bug0 Triage Checklist ([Main S/O](https://stackoverflowteams.com/c/expensify/questions/14418))

- [x] This "bug" occurs on a supported platform (ensure `Platforms` in OP are ✅)
- [x] This bug is not a duplicate report (check [E/App issues](https://github.com/Expensify/App/issues) and [#expensify-bugs](https://expensify.slack.com/archives/C049HHMV9SM))
    - If it is, comment with a link to the original report, close the issue and add any novel details to the original issue instead
- [x] This bug is reproducible using the reproduction steps in the OP. [S/O](https://stackoverflowteams.com/c/expensify/questions/14418#:~:text=The%20bug%20is%20reproducible)
    - If the reproduction steps are clear and you're unable to reproduce the bug, check with the reporter and QA first, then close the issue.
    - If the reproduction steps aren't clear and you determine the correct steps, please update the OP.
- [x] This issue is filled out as thoroughly and clearly as possible
    - Pay special attention to the title, results, platforms where the bug occurs, and if the bug happens on staging/production.
- [x] I have reviewed and subscribed to the linked Slack conversation to ensure Slack/Github stay in sync

**Comment by davidcardoza** [comment]
My fault, I accidentally clicked the `external` button in GH

@marcochavezf what needs to happen next?

**Comment by marcochavezf** [comment]
Hmm I wonder why the BugZero checklist didn't appear, I suppose because the linked PR doesn't correspond to the App repo. Pasting the BugZero checklist here:

**BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:

    - [ ] [@parasharrajat / @marcochavezf] The PR that introduced the bug has been identified. Link to the PR:
    - [ ] [@parasharrajat / @marcochavezf] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:
    - [ ] [@parasharrajat / @marcochavezf] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:
    - [ ] [@parasharrajat / @marcochavezf] Propose [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps to ensure the same bug will not reach production again.
    - [ ] [@davidcardoza] Link the GH issue for creating/updating the regression test once above steps have been agreed upon:

**Comment by marcochavezf** [comment]
Proposed regression test steps:

1. Create a Workspace chat (Go to Settings > Workspaces > New workspace)
2. Delete it
3. Open the archived workspace chat 
4. Verify the message "You closed this report" doesn't appear (just the footer explaining why the workspace chate is no longer active).

**Comment by davidcardoza** [comment]
Discussing here - https://expensify.slack.com/archives/C01SKUP7QR0/p1676681262825179

**Comment by davidcardoza** [comment]
Done

**Comment by MelvinBot** [comment]
@davidcardoza, @marcochavezf, @parasharrajat Uh oh! This issue is overdue by 2 days. Don't forget to update your issues!

**Comment by davidcardoza** [comment]
No regression tests are needed to be updated in testrail. Closing.
